pkg/minishift/registration/redhat.go

```diff
@@ -67,7 +74,16 @@ func (registrator *RedHatRegistrator) Register(param *RegistrationParameters) er
 			subscriptionCommand := fmt.Sprintf("sudo -E subscription-manager register --auto-attach "+
 				"--username %s "+
 				"--password '%s' ", param.Username, minishiftStrings.EscapeSingleQuote(param.Password))
+			spinnerView.Start()
+			startTime := time.Now()
 			_, err = registrator.SSHCommand(subscriptionCommand)
+			spinnerView.Stop()
+			if err == nil {
+				fmt.Print("Registration successful ")
+			} else {
+				fmt.Print("Registration unsuccessful ")
+			}
+			util.TimeTrack(startTime, os.Stdout)
```

Comment:
Regarding the format, we are displaying too many decimals. I think the problem is the use of %s in Sprintf call in TimeTrack. It might be all problems solved when we use %v. Have a look at https://www.goinggo.net/2013/06/gos-duration-type-unravelled.html as well.